projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -83,11 +83,13 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     protected readonly groupId?: string
   ) {}
 
-  public showCrosshair(locationData: MouseLocationData<TData, Series<TData> | Band<TData>>[]): void {
-    if (locationData.length > 0) {
+  public showCrosshair(locationData: MouseLocationData<TData, Series<TData> | Band<TData>>[] | undefined): void {
```

Comment:
calling showCrosshair(undefined) to hide a crosshair is counter intuitive. Instead, please expose hideCrosshair.